src/mdns/mdns_mdnssd.cpp

```diff
@@ -227,9 +261,11 @@ void PublisherMDnsSd::UpdateFdSet(fd_set & aReadFdSet,
     (void)aErrorFdSet;
     (void)aTimeout;
```

Comment:
Not a part of this PR, but prefer to use `OTBR_UNUSED_VARIABLE`.

Reply:
sure, I will create another PR to fix this.

Reply:
created PR https://github.com/openthread/ot-br-posix/pull/695 for this.